docs/README.md

```diff
@@ -596,21 +602,48 @@ GET /../../etc/password HTTP/1.1
 
 ## Request items
 
-There are a few different *request item* types that provide a convenient mechanism for specifying HTTP headers, simple JSON and form data, files, and URL parameters.
+There are a few different *request item* types that provide a convenient
+mechanism for specifying HTTP headers, JSON and form data, files,
+and URL parameters. This is a very practical way of constructing
+HTTP requests from scratch on the CLI.
+
+Each *request item* is simply a key/value pair separated with the following
+characters: `:` (headers), `=` (data field, e.g JSON, Form), `:=` (raw data field)
+`==` (query parameters), `@` (file upload).
 
-They are key/value pairs specified after the URL. All have in common that they become part of the actual request that is sent and that their type is distinguished only by the separator used: `:`, `=`, `:=`, `==`, `@`, `=@`, `:=@` and `==@`. The ones with an `@` expect a file path as value.
+```bash
+$ http PUT pie.dev/put \
+    X-Date:today \                     # Header
+    token==secret \                    # Query parameter
+    name=John \                        # String (default)
```

Comment:
`String (default)` doesn’t explain what it does

Reply:
The space before the comments don’t need to be to so big

Reply:
Perhaps
```suggestion
    name=John \                        # String Data (JSON or Form)
```

Reply:
The table below describes the concepts so labels should be consistent with that. This is a data field.